src/libraries/System.Reflection.Emit/tests/PersistedAssemblyBuilder/AssemblySaveTypeBuilderTests.cs

```diff
@@ -989,11 +996,17 @@ public class ClassWithFields : EmptyTestClass
         public byte field2;
     }
 
-    public unsafe class ClassWithFunctionPointerFields
+    public unsafe class ClassWithFunctionPointerMembers
     {
-        public static delegate*<ClassWithFunctionPointerFields> field1;
+        public static delegate*<ClassWithFunctionPointerMembers> field1;
         public static delegate* unmanaged<int> field2;
         public static delegate* unmanaged[Cdecl]<Guid> field3;
         public static delegate* unmanaged[Cdecl, SuppressGCTransition]<Vector<int>, Vector<int>> field4;
```

Comment:
```suggestion
        public static delegate* unmanaged[Cdecl, SuppressGCTransition]<Vector<int>, Vector<int>> field4;
        public static List<delegate* unmanaged[Stdcall, MemberFunction, SuppressGCTransition]<long>*[]> field5;
```
Could you please also add some complex case like List of arrays of pointers to a function pointer?